crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/space.rs

```diff
@@ -13,7 +13,7 @@ const DEFAULT_NOTIFY_TIMEOUT_MS: u64 = 1000;
 const DEFAULT_RPC_MULTI_REMOTE_AGENT_COUNT: u8 = 2;
 
 /// if the user specifies None or zero (0) for timeout_ms
-const DEFAULT_RPC_MULTI_TIMEOUT_MS: u64 = 1000;
+const DEFAULT_RPC_MULTI_TIMEOUT_MS: u64 = 20;
```

Comment:
erm, 20 ms is not reasonable for a network request - this might actually need to be on the level of multiple seconds.

Reply:
Because this isn't actually changing the timeout I've made a separate pr for this so we can test it #464